dashboard/modules/job/common.py

```diff
@@ -238,3 +241,77 @@ class JobStopResponse:
 @dataclass
 class JobLogsResponse:
     logs: str
+
+
+class DriverInfo(BaseModel):
+    """A class for recording information about the driver related to the job."""
+
+    id: str = Field(..., description="The id of the driver")
+    ip_address: str = Field(
+        ..., description="The ip address of the node the driver is running on"
+    )
```

Comment:
should we use node_id additionally / instead of ip address?

Reply:
i can add the node_id as a separate field but the ip address is actually useful for the oss dashboard UI because it's required to link to the logs page for specific nodes.

Reply:
I decided not to add `node_id` for now. It's not easily accessible in the `jobs_table_entry protobuf`.